README.md

```diff
@@ -43,7 +43,7 @@ The technical architecture of Apache APISIX:
 
 - Mailing List: Mail to dev-subscribe@apisix.apache.org, follow the reply to subscribe to the mailing list.
 - QQ Group - 552030619, 781365357
-- Slack Workspace - join [`#apisix` channel](https://join.slack.com/t/the-asf/shared_invite/zt-vlfbf7ch-HkbNHiU_uDlcH_RvaHv9gQ) on ASF Slack to meet the team and ask questions, or send `Request to join APISIX slack` mail to the mail list(dev@apisix.apache.org), we will invite you in.
+- Slack Workspace - join [`#apisix` channel](https://apisix.apache.org/docs/general/join/#join-the-slack-channel)
```

Comment:
@juzhiyuan Instead of jumping again, you can paste content directly here. One more jump, one more loss